<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>619</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>849</width>
    <height>619</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>849</width>
    <height>619</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SNAKE GAME: GAME</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>811</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>570</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>570</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNotify">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>761</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>You win!</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSnakeLenght">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Snake lenght: 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimePassed">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time elapsed: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGameScore">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Game score: 0</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>pushButtonReset</zorder>
  <zorder>pushButtonPause</zorder>
  <zorder>labelNotify</zorder>
  <zorder>labelSnakeLenght</zorder>
  <zorder>labelGameScore</zorder>
  <zorder>labelTimePassed</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
